cultcargo/genesis/casa/casa-inputs.yml

```diff
@@ -33,3 +33,9 @@ lib:
                 image: 
                     _use: vars.cult-cargo.images
                     name: casa6
+                backend:
+                    singularity:
+                        bind_dirs:
+                            dot-casa:
```

Comment:
Is `dot-*` a special construct for hidden files/folders?

Reply:
No, the label is arbitrary.

Reply:
See https://github.com/caracal-pipeline/stimela/issues/377#issuecomment-2751627645

Reply:
Yeah - I recall that change. Was just wondering if there was some meaning to the label I wasn't understanding.